### ISSUE TITLE
quickJump shortcut `=` broken - should bring up most recent folders

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#382]
A regression in Thunderbird 115 - using the special keystroke `=` would show a list of folders that were most recently used by the function. at the moment there is no visible reaction on screen.

In error console, we see the following message:

```log
[JavaScript Warning: "Exception in addSubFoldersPopupFromList: 
subfolder.getNumUnread is not a function" {file: "chrome://quickfolders/content/quickfolders-interface.js" line: 4538 column: 0 source: "addSubFoldersPopupFromList
  chrome://quickfolders/content/quickfolders-interface.js:4538:31
createRecentPopup@chrome://quickfolders/content/quickfolders-interface.js:297:8
findFolderName@chrome://quickfolders/content/quickfolders-interface.js:5137:46
initDelayed/<@chrome://quickfolders/content/quickfolders.js:538:16
"}] 14 quickfolders-util.js:1318:15

```

**Comment by RealRaven2000** [comment]
Here is a quick fix:

[QuickFolders-mx-6.1pre52.zip](https://github.com/RealRaven2000/QuickFolders/files/12239896/QuickFolders-mx-6.1pre52.zip)

---

To test the version above, download the zip file, drag the file into Thunderbird Add-ons Manager, do not extract contents or if won't install.

**Comment by aarnolds** [comment]
Perfect, fixed! Thanks for you very fast revision.

**Comment by RealRaven2000** [comment]
Fixed in 6.1 (published 08/08/2023)
